### PR TITLE
test/e2e - Add e2e test for Realtime Kernel

### PR DESCRIPTION
**Pull request by sinnykumari**
Attempt to include e2e test for kernelType

### CODE REVIEW

**Comment by sinnykumari** [comment]
e2e gcp is giving `msg="Error: Error waiting to create Subnetwork: Error waiting for Creating Subnetwork: Quota 'SUBNETWORKS' exceeded. Limit: 450.0 globally` . I think it's better to wait for getting quota fixed first

File: test/e2e/mcd_test.go [hunk 1]
```diff
@@ -283,6 +283,47 @@ func TestKernelArguments(t *testing.T) {
...
+		mcdName := mcd.ObjectMeta.Name
+		// Worker node should have RT kernel
+		cmd, err := exec.Command("oc", "rsh", "-n", "openshift-machine-config-operator", mcdName,
+			"uname", "-a").CombinedOutput()
+		require.Nil(t, err)
+		kernelInfo := string(cmd)
+		if !strings.Contains(kernelInfo, "PREEMPT RT") {
+			t.Fatalf("Node %s doesn't have expected kernel", node.Name)
+		}
+		t.Logf("Node %s has expected kernl", node.Name)
```
Comment (kikisdeliveryservice) [inline test/e2e/mcd_test.go]: kernl -> kernel

Comment (sinnykumari) [reply test/e2e/mcd_test.go]: fixed

**Comment by kikisdeliveryservice** [comment]
This looks good, just waiting for the confirmation via gcp-op

**Comment by sinnykumari** [comment]
Thanks Kirsten for review!

File: test/e2e/mcd_test.go [hunk 2]
```diff
@@ -283,6 +283,47 @@ func TestKernelArguments(t *testing.T) {
...
+		// Worker node should have RT kernel
+		cmd, err := exec.Command("oc", "rsh", "-n", "openshift-machine-config-operator", mcdName,
+			"uname", "-a").CombinedOutput()
+		require.Nil(t, err)
+		kernelInfo := string(cmd)
+		if !strings.Contains(kernelInfo, "PREEMPT RT") {
+			t.Fatalf("Node %s doesn't have expected kernel", node.Name)
+		}
+		t.Logf("Node %s has expected kernel", node.Name)
+	}
```
Comment (runcom) [inline test/e2e/mcd_test.go]: I think we should roll back to default to make this test more effective also and verify that works as well @sinnykumari

Comment (sinnykumari) [reply test/e2e/mcd_test.go]: make sense, I will open a new PR for it

**Pull request merged**